flink-yarn/src/main/java/org/apache/flink/yarn/YarnLocalResourceDescription.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.yarn;
+
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.yarn.api.records.LocalResourceVisibility;
+
+import javax.annotation.Nonnull;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Yarn local resource description is generated by {@link YarnClusterDescriptor} and set to environment of job manager.
+ * Then it will be used to register local resources for task manager in {@link Utils#createTaskExecutorContext}.
+ */
+class YarnLocalResourceDescription {
```

Comment:
maybe `YarnLocalResourceDescriptor`? **Descriptor** makes more sense to me

Reply:
Make sense to me. I will update.